include/libopencm3/stm32/g4/rcc.h

```diff
@@ -7,15 +7,34 @@
  *
  * @version 1.0.0
  *
- * @author @htmlonly &copy; @endhtmlonly 2020 Karl Palsson <karlp@tweak.net.au>
+ * @author @htmlonly &copy; @endhtmlonly 2009
+ * Federico Ruiz-Ugalde \<memeruiz at gmail dot com\>
+ * @author @htmlonly &copy; @endhtmlonly 2009
+ * Uwe Hermann <uwe@hermann-uwe.de>
+ * @author @htmlonly &copy; @endhtmlonly 2011
+ * Fergus Noble <fergusnoble@gmail.com>
+ * @author @htmlonly &copy; @endhtmlonly 2011
+ * Stephen Caudle <scaudle@doceme.com>
+ * @author @htmlonly &copy; @endhtmlonly 2020
+ * Karl Palsson <karlp@tweak.net.au>
+ * @author @htmlonly &copy; @endhtmlonly 2020
+ * Sam Kirkham <sam.kirkham@codethink.co.uk>
+ * @author @htmlonly &copy; @endhtmlonly 2020
+ * Ben Brewer <ben.brewer@codethink.co.uk>
  *
  * LGPL License Terms @ref lgpl_license
  */
 
 /*
  * This file is part of the libopencm3 project.
  *
+ * Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+ * Copyright (C) 2009 Federico Ruiz-Ugalde <memeruiz at gmail dot com>
+ * Copyright (C) 2011 Fergus Noble <fergusnoble@gmail.com>
+ * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
  * Copyright (C) 2020 Karl Palsson <karlp@tweak.net.au>
+ * Copyright (C) 2020 Sam Kirkham <sam.kirkham@codethink.co.uk>
+ * Copyright (C) 2020 Ben Brewer <ben.brewer@codethink.co.uk>
```

Comment:
so run it by me again, I made a new file from scratch, you (ben? sam?) added to it, and have added 4 new authors back to 2009?  I.... struggle to follow how that works.

Reply:
The new file is based on a direct copy of an existing file, from my limited understanding that doesn't remove copyright, but if that's not the process for adding copyright to the files then I'm happy to change this.

Reply:
Ah sorry, yeah I included authors from any file I copied directly from too, I guess that's incorrect? Should I just change this to the direct authors of the new content in the file?